src/main/webapp/i18n/de/consistencyCheck.json

```diff
@@ -10,12 +10,15 @@
                 "VCS_PROJECT_MISSING": "Fehlendes VCS Projekt",
                 "TEMPLATE_REPO_MISSING": "Fehlendes Vorlagen-Repository",
                 "SOLUTION_REPO_MISSING": "Fehlendes Lösung-Repository",
+                "AUXILIARY_REPO_MISSING": "Fehlendes Hilfs-Repository",
                 "TEST_REPO_MISSING": "Fehlendes Test-Repository",
                 "TEMPLATE_BUILD_PLAN_MISSING": "Fehlender Build-Plan für die Vorlage",
-                "SOLUTION_BUILD_PLAN_MISSING": "Fehlender Build-Plan für die Lösung"
+                "SOLUTION_BUILD_PLAN_MISSING": "Fehlender Build-Plan für die Lösung",
+                "programmingExerciseImportFailed": "Importieren der Programmieraufgabe fehlgeschlagen. Quellaufgabe ist inkonsistent."
```

Comment:
This sounds very abstract. Without the context of this PR I would not be sure what this means and how to solve the issue.